### DISCUSSION TITLE
I apply for student pack

### DISCUSSION THREAD

**Discussion by ArkitPatel123** [community/community#56188] [GitHub Education]
### Select Topic Area

Bug

### Body

well I am a student and I apply for GitHub Student pack to access GitHub copilot but my application was continuously rejected why?

**Comment by samus-aran** [comment]
Hi @ArkitPatel123  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
